assets/i18n/en.json

```diff
@@ -680,6 +680,7 @@
     "I18N_INTERACTIONS_IMAGE_CLICK_INSTRUCTION": "Click on the image",
     "I18N_INTERACTIONS_IMAGE_CLICK_SELECT": "[Select an image to display]",
     "I18N_INTERACTIONS_INPUT_NO_RESPONSE": "Enter an answer to continue",
+    "I18N_INTERACTIONS_ITEM_SELECTION_EXACT": "Please select exactly {minChoiceNumber} choices.",
```

Comment:
This is not `minChoiceNumber`. Please replace it with something like `exactChoiceNumber` or `choiceNumber`

Reply:
Updated to exactChoiceNumber